api/platform/types.go

```diff
@@ -1508,3 +1508,44 @@ type ClusterGroupAPIResourceItem struct {
 type ClusterGroupAPIResourceOptions struct {
 	metav1.TypeMeta
 }
+
+type OverrideValue struct {
+	Global                       Global                       `json:"global"`
+	NfsSubdirExternalProvisioner NfsSubdirExternalProvisioner `json:"nfs-subdir-external-provisioner"`
+	CephCsiCephfs                CephCsiCephfs                `json:"ceph-csi-cephfs"`
+}
+type Global struct {
+	EnableNFS    bool `json:"enableNFS"`
+	EnableCephFS bool `json:"enableCephFS"`
+}
+type Nfs struct {
+	Server string `json:"server"`
+	Path   string `json:"path"`
+}
+type NfsSubdirExternalProvisioner struct {
+	Nfs Nfs `json:"nfs"`
+}
+type CsiConfig struct {
+	ClusterID string   `json:"clusterID"`
+	Monitors  []string `json:"monitors"`
+}
+type StorageClass struct {
+	ClusterID string `json:"clusterID"`
```

Comment:
if set struct in types, we should set the struct without `json:xxx` in /api/platform, and the struct with `json:xxx` in /api/v1/platform.